pkg/image/image.go

```diff
@@ -29,7 +29,7 @@ const (
 
 const (
 	// RootFs partition name
-	RootFs       = "rootfs"
+	RootFs       = "!__rootfs__!"
```

Comment:
Everything looks good.

The one thing I'm not sure about is this change.

Does this have an impact on the images that get created? In particular, what happens with images created before this change? I see there's a new check using HasRootFs... are older images rejected?

Related to this: does the change on line 21 (SQUASHFS) have an impact on images?

Reply:
No impact, those informations are used by runtime only to somewhat abstract image format